### PR TITLE
[14.0][MIG] website_sale_product_brand: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Please @chienandalu  and @joao-p-marques can you review it?

@Tecnativa TT31863

### CODE REVIEW

File: website_sale_product_brand/README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to install following module:
+->https://github.com/OCA/product-attribute/tree/12.0/product_brand
```
Comment (chienandalu) [inline website_sale_product_brand/README.rst]: Update this reference or maybe remove it as it's implicit in the dependencies

**Review by chienandalu — APPROVED** [review]
Just a tiny comment

**Review by joao-p-marques — APPROVED** [review]
Code review. LGTM :+1:

**Pull request merged**